qrenderdoc/Windows/PipelineState/VulkanPipelineStateViewer.cpp

```diff
@@ -4368,6 +4395,12 @@ QVariant VulkanPipelineStateViewer::ConvertSDObjectToFossilizeJSON(const SDObjec
 
       AddFossilizeNexts(map, obj);
 
+      if(map.contains(lit("sampleMask")))
+      {
+        auto sampleMaskArray = QVariantList({map[lit("sampleMask")]});
```

Comment:
Minor nit, please don't use `auto` except for things like lambdas or stl iterators. `QVariantList` here is preferred to `auto`.